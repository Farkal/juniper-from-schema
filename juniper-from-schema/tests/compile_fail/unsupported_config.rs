#![allow(dead_code, unused_variables, unused_must_use, unused_imports)]
include!("../setup.rs");

graphql_schema_from_file!(
    "../../../juniper-from-schema/tests/schemas/customizing_context_name.graphql",
    foo: Foo
);

pub struct Query;

impl QueryFields for Query {
    fn field_foo<'a>(&self, _: &Executor<'a, Context>) -> FieldResult<&String> {
        unimplemented!()
    }
}
