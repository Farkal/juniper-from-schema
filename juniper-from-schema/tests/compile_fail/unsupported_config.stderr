error: Unknown `graphql_schema_from_file` config `foo`
Supported configs are `error_type` and `context_type`

 --> $DIR/unsupported_config.rs:6:5
  |
6 |     foo: Foo
  |     ^^^

error[E0405]: cannot find trait `QueryFields` in this scope
  --> $DIR/unsupported_config.rs:11:6
   |
11 | impl QueryFields for Query {
   |      ^^^^^^^^^^^ not found in this scope

For more information about this error, try `rustc --explain E0405`.
error: Could not compile `juniper-from-schema-tests`.
